app/helpers/application_helper/form_tags.rb

```diff
@@ -1,5 +1,15 @@
 module ApplicationHelper
   module FormTags
+    # override: default password_field_tag doesn't switch off autocomplete
+    def password_field_tag(name, value = nil, options = {})
+      options = {
+        :type => 'password',
+        :autocomplete => 'off',
+      }.merge(options)
+
+     text_field_tag(name, value, options)
```

Comment:
2 space indent is incorrect here

Reply:
@himdel Can you fix the 2 rubocops on the ruby code?

Reply:
Oh, yeah, 1 space is not enough :)

Should be fixed :)